Packs/Cryptosim/Integrations/Cryptosim/Cryptosim.py

```diff
@@ -0,0 +1,251 @@
+
+    for inc in incident_data:
+
+        if len(incidentsList) > max_results:
+            break
+
+        incident_name = inc['CorrelationAlert']['NAME']
+        time_stamp = inc['CorrelationAlert']['CREATEDATE'] + "Z"
+
+        severity_level = int(inc['CorrelationAlert']['RISK'])
+        if severity_level >= 0 and severity_level <= 5:
+            severity = 1
+        elif severity_level > 5 and severity_level <= 7:
+            severity = 2
+        elif severity_level > 7 and severity_level <= 9:
+            severity = 3
+        elif severity_level > 9 and severity_level <= 10:
+            severity = 4
+        else:
+            severity = 0
+
+        # "log" column is stringfyed 'Log' data.
+        inc['Log'].pop("log")
+
+        incident_object = {**inc['Log'], **inc['CorrelationAlert']}
+
+        incident = {
+            'name': incident_name,
+            'occurred': time_stamp,
+            'rawJSON': json.dumps(incident_object),
+            "severity": severity,
+            'type': 'Crpyotsim CorrelationAlert'
+        }
+
+        incidentsList.append(incident)
+
+        created_incident = datetime.strptime(time_stamp, DATE_FORMAT)
+        last_fetch = datetime.strptime(last_fetch, DATE_FORMAT)
+        if created_incident > last_fetch:
+
+            last_fetch = created_incident + timedelta(milliseconds=10)
+
+    # Save the next_run as a dict with the last_fetch key to be stored
+    next_run = {'last_fetch': last_fetch}
+    return next_run, incidentsList
+
+
+''' HELPERS '''
+
+
+def get_client(params):
+    authorization = params.get('credentials').get(
+        'identifier') + ":" + params.get('credentials').get('password')
+    auth_byte = authorization.encode('utf-8')
+    base64_byte = base64.b64encode(auth_byte)
+    base64_auth = base64_byte.decode('utf-8')
+    authValue = "Basic " + base64_auth
+
+    headers = {
+        "Content-Type": "application/json",
+        'Authorization': authValue
+    }
+    # get the service API url
+    base_url = urljoin(params.get('url'), '/api/service/')
+    proxy = params.get('proxy', False)
+
+    client = Client(
+        base_url=base_url,
+        verify=False,
+        headers=headers,
+        proxy=proxy)
+    return client
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    params = demisto.params()
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        client = get_client(params)
+
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            return_results(result)
+
+        elif demisto.command() == 'cryptosim-get-correlations':
+            return_results(correlations_command(client))
+
+        elif demisto.command() == 'cryptosim-get-correlation-alerts':
```

Comment:
I see you added to "get-correlations" limit. Is it not possible to add also to "get-correlation-alerts"?

Reply:
I added both of them. We will update end points, now limit can be given as argument.